edbdeploy/project.py

```diff
@@ -691,6 +691,10 @@ def deploy(self, no_install_collection,
         if self.cloud == 'azure-db':
             extra_vars.update(dict(azure_db_hackery=True))
 
+        if self.cloud == 'vmware':
+            # Set SSH password
+            extra_vars.update(dict(ansible_ssh_pass='vagrant'))
+
```

Comment:
This should be done in the `_build_ansible_vars()` method of the `VMwareProject` class.

Reply:
I added it to the declaration of the extra_vars